### PR TITLE
Add Korean translation for v2.0

### PR DESCRIPTION
**Pull request by estherk0**
PR for new Korean translation of v2.0. 
I could find some Korean natives for the review if you don't mind.

### CODE REVIEW

File: content/version/2/0/code-of-conduct.ko.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+커뮤니티의 긍정적인 환경을 위해 기여자가 해야 할 행동은 다음과 같습니다:
+
+* 소외하지 않고 배려하는 언어 사용
+* 서로 다른 경험과 관점 존중
+* 열린 마음으로 건설적인 비판을 수용
+* 커뮤니티에 가장 최선이 무엇인지에 주력
+* 다른 커뮤니티 구성원들에 대한 공감 표현
+
+* 다른 사람들에 대한 친절과 공감 표현
+* 다양한 의견, 관점 그리고 경험을 존중
```
Comment (seokho-son) [inline content/version/2/0/code-of-conduct.ko.md]: @jabbukka I cannot find this section L22~L27 from the English version. Maybe outdated contents :)

```suggestion
커뮤니티의 긍정적인 환경을 위해 기여자가 해야 할 행동은 다음과 같습니다:

* 다른 사람들에 대한 친절과 공감 표현
* 다양한 의견, 관점 그리고 경험을 존중
```
https://github.com/ContributorCovenant/contributor_covenant/blob/release/content/version/2/0/code_of_conduct.md
```
Examples of behavior that contributes to a positive environment for our
community include:

* Demonstrating empathy and kindness toward other people
* Being respectful of differing opinions, viewpoints, and experiences
* Giving and gracefully accepting constructive feedback
* Accepting responsibility and apologizing to those affected by our mistakes,
  and learning from the experience
* Focusing on what is best not just for us as individuals, but for the
  overall community
```

Comment (estherk0) [reply content/version/2/0/code-of-conduct.ko.md]: Corrected! Thanks.

**Review by bluejayA — COMMENTED** [review]
Throughout this document, honorific words such as _"같습니다."_ are used. I would suggest we use non honorific words, such as _"같다"_ , in this translation. 

Since we usually use non honorific words in "declaration document" in Korea (in my experience), IMHO, following "declaration document" pattern would  be better for this translation. 

For instance, instead of "행동은 다음과 같습니다", use "행동은 다음과 같다".

**Comment by bluejayA** [comment]
@seokho-son It would be great if I can get your opinion on my review comment, regarding use of honorific words.

File: content/version/2/0/code-of-conduct.ko.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
++++
+
+# 기여자 행동 강령 규약
+
+## 서약
+
+구성원, 기여자 및 리더로서 커뮤니티에서 연령, 신체 크기,
+눈에 보이거나 보이지 않는 장애, 민족성, 성별, 성 정체성과 표현,
+경력, 학력, 사회 경제적 지위, 국적, 외모, 인종, 종교
+또는 성적 정체성과 지향에 관계없이 모두에게 차별없이 참여할 것을 서약합니다.
```
Comment (seungkyua) [inline content/version/2/0/code-of-conduct.ko.md]:
```suggestion
우리는 구성원, 기여자 및 리더로서 커뮤니티에 참여하여
연령, 신체 크기, 눈에 보이거나 보이지 않는 장애, 민족성, 성별, 성 정체성과 표현,
경력, 학력, 사회 경제적 지위, 국적, 외모, 인종, 종교
또는 성적 정체성과 성적 성향에 관계없이 모든 사람을 차별하지 않을 것을 서약한다.
```

File: content/version/2/0/code-of-conduct.ko.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+개방적이고 친근하며 다양하고 포용적이며 건강한 커뮤니티에 기여하는
+방식으로 행동하고 상호작용할 것을 서약합니다.
```
Comment (seungkyua) [inline content/version/2/0/code-of-conduct.ko.md]:
```suggestion
우리는 개방적이고 친근하며 다양하고 포용적이며 건강한 커뮤니티에 기여하는
방식으로 행동하고 상호작용할 것을 서약한다.
```

File: content/version/2/0/code-of-conduct.ko.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+## 표준
+
+커뮤니티의 긍정적인 환경을 위해 기여자가 해야 할 행동은 다음과 같습니다:
+
+* 다른 사람들에 대한 친절과 공감 표현
+* 다양한 의견, 관점 그리고 경험을 존중
```
Comment (seungkyua) [inline content/version/2/0/code-of-conduct.ko.md]:
```suggestion
* 서로 다른 의견 및 관점, 경험에 대한 존중
```

**Comment by estherk0** [comment]
@bluejayKR @seokho-son @seungkyua I've applied all feedbacks.

**Review by bluejayA — APPROVED** [review]
LGTM

**Comment by seokho-son** [comment]
> @seokho-son It would be great if I can get your opinion on my review comment, regarding use of honorific words.

I don't have a strong opinion on this subject :) 

I agree with using non-honorific (polite) words of Korean is appropriate especially for a document that requires clarity such as technical / legal documents. Considering "Code of Conduct" doc is for a declaration, using non-honorific words for Korean translation enhance clarity.

On the other hand, using honorific words is useful to friendly welcome paricipants and readers. I think "Code of Conduct" is a document for welcoming participants as well. So, I think using honorific words is also benificial for this document. 
(I found several cases that Korean code of conduct documents using honorific words by [web searching](https://www.google.co.kr/search?dcr=0&sxsrf=ALeKk01BZnGK4HF2Zo5DaFwlRAQ_CKkQDA%3A1610002063587&source=hp&ei=j672X_usIa7xhwPxs704&q=code+of+conduct+%ED%96%89%EB%8F%99+%EA%B0%95%EB%A0%B9&oq=code+o&gs_lcp=CgZwc3ktYWIQAxgBMgQIIxAnMgQIIxAnMgcIABAUEIcCMgQIABAKMgcIABAUEIcCMgIIADICCAAyAggAMgUIABDLATICCAA6CAgAELEDEIMBOgUIABCxAzoHCCMQ6gIQJzoECAAQAzoECAAQQzoKCAAQsQMQFBCHAjoNCAAQsQMQgwEQFBCHAlC6CFiaHWDrKGgDcAB4AIABhgGIAbEGkgEDMC43mAEAoAEBqgEHZ3dzLXdperABCg&sclient=psy-ab))

File: content/version/2/0/code-of-conduct.ko.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+이 행동 강령은 [기여자 규약][homepage] 의 2.0 버전을 변형하였습니다. 그 내용은
+[https://www.contributor-covenant.org/ko/version/2/0/code-of-conduct.html][v2.0] 에서
+확인할 수 있습니다.
+
+커뮤니티 영향 지침은 [Mozilla's code of conduct enforcement ladder][Mozilla CoC]에서 
+영감을 얻었습니다.
+
+이 행동 강령에 관련한 일반적인 질문에 대한 대답은 [https://www.contributor-covenant.org/faq][FAQ]를
+참고할 수 있습니다. 번역본은 [https://www.contributor-covenant.org/translations][translations]에서 볼 수 있습니다.
```
Comment (seokho-son) [inline content/version/2/0/code-of-conduct.ko.md:121]: For consistency,

```suggestion
이 행동 강령은 [기여자 규약][homepage] 의 2.0 버전을 변형한 것이다. 그 내용은
[https://www.contributor-covenant.org/ko/version/2/0/code-of-conduct.html][v2.0] 에서
확인 가능하다.

커뮤니티 영향에 대한 지침은 [Mozilla's code of conduct enforcement ladder][Mozilla CoC]에서 
영감을 얻어 활용하였다.

이 행동 강령에 관련한 일반적인 질문에 대한 대답은 [https://www.contributor-covenant.org/faq][FAQ]를
참고 가능하다. 번역본은 [https://www.contributor-covenant.org/translations][translations]에서 볼 수 있다.
```

**Review by seungkyua — COMMENTED** [review]
"~한다" 체로 바꾸는 것이 좋을 것 같습니다.

File: content/version/2/0/code-of-conduct.ko.md (hunk 5 above)
Comment (bluejayA) [reply content/version/2/0/code-of-conduct.ko.md:121]: IMHO, sentences under "참고" section are not the main text of "code of conduct", it is rather additional content describing /explaining a supplement information. In that sense, I thought having "horonofic" in this "참고" section would be better. 

(참고에 해당하는 내용은 읽는 분들에게 추가 정보를 알려주는 문구라, 경어체를 쓰도 일관성을 해치지 않는다고 생각했습니다)

**Comment by estherk0** [comment]
> "~한다" 체로 바꾸는 것이 좋을 것 같습니다.

적용되었습니다.

**Review by bluejayA — APPROVED** [review]
LGTM

**Comment by seungkyua** [comment]
LGTM

**Comment by bluejayA** [comment]
@CoralineAda Cloud you please look at this PR, and guide us how to proceed from here? Thanks!

**Comment by CoralineAda** [comment]
@bluejayKR If you've addressed all the relevant comments, are you ready for me to merge?

**Comment by estherk0** [comment]
@CoralineAda I've resolved comments. I think we are ready to merge!

**Comment by CoralineAda** [comment]
Thank you @jabbukka and all the reviewers— amazing work!

**Pull request merged**